lib/core/store.ts

```diff
@@ -31,6 +33,17 @@ export function isSearchPromiseResult(obj: any): obj is SearchPromiseResult {
   );
 }
 
+export function isSearchSchedulesResp(obj: any): obj is { cursor: string; schedules: Schedule[] } {
```

Comment:
Can we align the names to end in either "resp" or "result". I'm easy on which one, just want it to be consistent.